### PR TITLE
more css fixes

### LINKED ISSUE #180: Dashboard css is applied before static css
Hi,

Dashboard css is applied before some static css this forces user to add !important hint. Can it be applied the latest?

image

Thanks

### PR DESCRIPTION
**Pull request by williaster**
@mistercrunch 
changes with this commit:
- fixoverflow scroll bars on all dashboard charts
- make dashboard chart control toggle interaction nicer (click to see instead of weird hover action)
- make sure user-stylesheets are applied last (fixes https://github.com/mistercrunch/panoramix/issues/180) 
- remove ace css editor error/warning parsing (it assumes it's the only stylesheet so warnings are inaccurate)
- make filters look nicer in dashboards
- fix some linting errors on master

will work on the modal z-index separately, wanted to get these in.

### CODE REVIEW

File: panoramix/assets/javascripts/dashboard.js [hunk 1]
```diff
@@ -149,7 +149,8 @@ var Dashboard = function (dashboardData) {
       editor.setTheme("ace/theme/crimson_editor");
       editor.setOptions({
         minLines: 16,
-        maxLines: Infinity
+        maxLines: Infinity,
+        useWorker: false
```
Comment (williaster) [inline panoramix/assets/javascripts/dashboard.js]: this is the flag for not watching errors.

File: panoramix/templates/panoramix/dashboard.html [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-      {{ dashboard.css or '' }}
-    </style>
+{% block head_js %}
+  {{ super() }}
+  <script src="/static/assets/javascripts/dist/dashboard.entry.js"></script>
 {% endblock %}
 
 {% block body %}
-<div class="dashboard container-fluid" data-dashboard="{{ dashboard.json_data }}">
+<div class="dashboard container-fluid" data-dashboard="{{ dashboard.json_data }}" data-css="{{ dashboard.css }}">
```
Comment (williaster) [inline panoramix/templates/panoramix/dashboard.html]: added the css here as data and created the `<style>` with JS

File: panoramix/assets/javascripts/dashboard.js [hunk 3]
```diff
@@ -170,19 +173,51 @@ var Dashboard = function (dashboardData) {
         gridster.remove_widget(li);
       });
 
-      $(".slice_info").click(function () {
-        var widget = $(this).parents('.widget');
-        var slice_description = widget.find('.slice_description');
-        slice_description.slideToggle(500, function () {
-          widget.find('.refresh').click();
-        });
+      $("li.widget").click(function (e) {
```
Comment (williaster) [inline panoramix/assets/javascripts/dashboard.js]: FYI this is typically better practice for event listeners, you place the listener on the parent element and the event "bubbles" up from the dom element you click on, up to the parent with the listener. then you don't have to add a million listeners.

**Comment by williaster** [comment]
hmm not sure what's up with the travis build, `npm install` failed.

File: panoramix/assets/javascripts/dashboard.js (hunk 3 above)
Comment (mistercrunch) [inline panoramix/assets/javascripts/dashboard.js]: Gotcha.

**Comment by mistercrunch** [comment]
Nice! LGTM

**Pull request merged**